packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Input/src/InputTypes/InputTypeSerialization.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Generator.CSharp.Input
+{
+    public sealed class InputTypeSerialization
+    {
+        public InputTypeSerialization(bool json, InputTypeXmlSerialization? xml, bool includeConverter)
+        {
+            Json = json;
+            Xml = xml;
+            IncludeConverter = includeConverter;
+        }
+
+        public bool Json { get; }
+        public InputTypeXmlSerialization? Xml { get; }
+        public bool IncludeConverter { get; }
```

Comment:
Is the converter only relevant for JSON?

Reply:
I took a look at autorest and it appears so. It also looks like it may only be used in the swagger scenario.. I removed it for now as it's not being used. We can pull it back in if it's needed.